Source/API/EbApi.h

```diff
@@ -10,6 +10,7 @@
 #include <string.h>
 #include <stdint.h>
 #include <EbApiVersion.h>
+#include <sys/queue.h>
```

Comment:
Does this header exist on windows and mac?

Reply:
No, they should use another library of doubly-linked list. The change is a prototype tried in Linux, and need to be also applied to Windows/Mac if it's accepted. Thanks!

Reply:
Seems that there is something similar for windows, but not exactly https://docs.microsoft.com/en-us/windows/win32/api/ntdef/ns-ntdef-list_entry

Reply:
Thanks! I checked it as well, and it seems that it's Windows kernel mode library, but not sure whether we could apply it in user mode. And I don't know the common one used in Mac...

Here is the clear [implementation](https://github.com/torvalds/linux/blob/master/include/linux/list.h) of doubly-linked list in Linux kernel.

BTW, I looked for such library in libavutil of ffmpeg, but didn't find it.